tests/sqllogictests/suites/query/window_function/window_basic.test

```diff
@@ -239,6 +239,92 @@ SELECT salary, lead(salary, 2, salary) OVER (ORDER BY enroll_date) FROM empsalar
 4200 4200
 4500 4500
 
+# first_value
+query TII
+SELECT enroll_date, salary, first_value(salary) OVER (ORDER BY enroll_date) FROM empsalary
+----
+2006-10-01 5000 5000
+2006-10-01 6000 5000
+2006-12-23 3900 5000
+2007-08-01 5200 5000
+2007-08-01 4800 5000
+2007-08-08 4800 5000
+2007-08-15 5200 5000
+2007-12-10 3500 5000
+2008-01-01 4200 5000
+2008-01-01 4500 5000
+
+query TII
+SELECT depname, salary, first_value(salary) OVER (PARTITION BY depname ORDER BY enroll_date) FROM empsalary
+----
+develop 6000 6000
+develop 5200 6000
+develop 5200 6000
+develop 4200 6000
+develop 4500 6000
+personnel 3900 3900
+personnel 3500 3900
+sales 5000 5000
+sales 4800 5000
+sales 4800 5000
+
+query TII
+SELECT depname, salary, first_value(salary) OVER (PARTITION BY depname ORDER BY salary) FROM empsalary
+----
+develop 4200 4200
+develop 4500 4200
+develop 5200 4200
+develop 5200 4200
+develop 6000 4200
+personnel 3500 3500
+personnel 3900 3500
+sales 4800 4800
+sales 4800 4800
+sales 5000 4800
+
+# last_value
+query TII
+SELECT enroll_date, salary, last_value(salary) OVER (ORDER BY enroll_date) FROM empsalary
+----
+2006-10-01 5000 6000
+2006-10-01 6000 6000
+2006-12-23 3900 3900
+2007-08-01 5200 4800
+2007-08-01 4800 4800
+2007-08-08 4800 4800
+2007-08-15 5200 5200
+2007-12-10 3500 3500
+2008-01-01 4200 4500
+2008-01-01 4500 4500
+
+query TII
+SELECT depname, salary, last_value(salary) OVER (PARTITION BY depname ORDER BY enroll_date) FROM empsalary
+----
+develop 6000 6000
+develop 5200 5200
+develop 5200 5200
+develop 4200 4500
+develop 4500 4500
+personnel 3900 3900
+personnel 3500 3500
+sales 5000 5000
+sales 4800 4800
+sales 4800 4800
+
+query TII
+SELECT depname, salary, last_value(salary) OVER (PARTITION BY depname ORDER BY salary) FROM empsalary
+----
+develop 4200 4200
+develop 4500 4500
+develop 5200 5200
+develop 5200 5200
+develop 6000 6000
+personnel 3500 3500
+personnel 3900 3900
+sales 4800 4800
+sales 4800 4800
+sales 5000 5000
```

Comment:
The expected results are not right.

Reply:
And can add the test cases in https://github.com/duckdb/duckdb/blob/master/test/sql/window/test_basic_window.test